extension/solarwindsapmsettingsextension/metadata.yaml

```diff
@@ -0,0 +1,9 @@
+type: solarwindsapmsettings
+
+status:
+  class: extension
+  stability:
+    development: [extension]
+  distributions: [contrib]
```

Comment:
```suggestion
  distributions: []
```

Reply:
Committed the suggested change, regenerated the READMD.md and I also added a yaml file under `.chloggen` so as to meet changelog github action requirement.